src/core_plugins/tests_bundle/tests_entry_template.js

```diff
@@ -26,7 +26,7 @@ window.__KBN__ = {
   vars: {
     kbnIndex: '.kibana',
     esShardTimeout: 1500,
-    esApiVersion: '2.0',
+    esApiVersion: 'master',
```

Comment:
I was thinking about creating a top level versions file in the root of the project that we can use to track all specific versions like this, so bumping the Kibana version would all be done in the same file.  That's separate from this PR, though.